public/controlpanel.php

```diff
@@ -279,9 +279,17 @@ function confirmEmailChange(event) {
             echo "<tr>";
             echo "<td class='align-top'>Web API Key</td>";
             echo "<td>";
-            echo "<input class='mb-1' size='60' readonly value='$apiKey'>";
+
+            $copyIcon = Blade::render('<x-fas-copy />');
 
             echo <<<HTML
+                <div role="button" x-init="{}" @click="copyToClipboard('$apiKey')" class="flex items-center gap-x-2 bg-embed max-w-fit rounded pl-2 mb-2">
```

Comment:
Is there a point to `x-init={}`?

Reply:
Yes. Without this directive, Alpine.js doesn't have an execution context for the component.